.eslintrc.json

```diff
@@ -15,7 +15,7 @@
 	"settings": { "react": { "version": "detect" } },
 	"rules": {
 		"no-duplicate-imports": "warn",
-		"no-unused-vars": "warn",
+		"no-unused-vars": ["warn", { "argsIgnorePattern": "^_" }],
```

Comment:
Curious about the reasoning behind this change! Are there specific cases where we're intentionally using unused variables or function parameters prefixed with an underscore